masterlist.yaml

```diff
@@ -22646,3 +22646,11 @@ plugins:
     clean:
       - crc: 0xBD12E363
         util: 'SSEEdit v4.0.4'
+
+  - name: 'WorldEaterBeater.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778' ]
+    after: [ 'DeadlyDragons.esp' ]
```

Comment:
LOOT should sort this way by default, but there's no harm in adding a hard rule just in case.